RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc

```diff
@@ -170,10 +183,14 @@ void PCCNTupler::analyze(const edm::Event& iEvent,
     // Get the Run, Lumi Section, and Event numbers, etc.
     runNo   = iEvent.id().run();
     LSNo    = iEvent.getLuminosityBlock().luminosityBlock();
-    LNNo    = -99; // FIXME need the luminibble
+    //LNNo    = -99; // FIXME need the luminibble
     eventNo = iEvent.id().event();
     bxNo    = iEvent.bunchCrossing();
-    timeStamp = iEvent.time().unixTime();
+    timeStamp_local = iEvent.time().unixTime();
+    if(timeStamp_end  <timeStamp_local) timeStamp_end   =timeStamp_local;
+    if(timeStamp_begin>timeStamp_local) timeStamp_begin =timeStamp_local;
+    orbitNo = iEvent.orbitNumber();
+    LNNo    = ((int) (orbitNo/pow(2,12)) % 64); //-99; // FIXME need the luminibble
```

Comment:
orbitNo is an int.  I didn't think bitwise operations would work
properly on a signed object, but it works fine.  I'll update this with
the rest.

On Wed, May 27, 2015 at 10:46 AM, Slava Krutelyov
<notifications@github.com> wrote:
> In RecoLuminosity/LumiProducer/test/analysis/plugins/PCCNTupler.cc:
>
>>      eventNo = iEvent.id().event();
>>      bxNo    = iEvent.bunchCrossing();
>> -    timeStamp = iEvent.time().unixTime();
>> +    timeStamp_local = iEvent.time().unixTime();
>> +    if(timeStamp_end  <timeStamp_local) timeStamp_end   =timeStamp_local;
>> +    if(timeStamp_begin>timeStamp_local) timeStamp_begin =timeStamp_local;
>> +    orbitNo = iEvent.orbitNumber();
>> +    LNNo    = ((int) (orbitNo/pow(2,12)) % 64); //-99; // FIXME need the
>> luminibble
>
> why not orbitNo >> 12 instead of pow(2,12)?
>
> —
> Reply to this email directly or view it on GitHub.